index.html

```diff
@@ -1016,7 +1021,7 @@ <h2><dfn>Coalesced events</dfn></h2>
                 original events' order. For example if a <code>pointerdown</code> event causes the dispatch for the
                 coalesced <code>pointermove</code> events the user agent SHOULD first dispatch one <code>pointermove</code>
                 event with all those coalesced events of a <code>pointerId</code> followed by the <code>pointerdown</code> event.
-                Here is an example of the actual events happening with increasing <code>timestamps</code> and the
+                Here is an example of the actual events happening with increasing {{Event/timeStamp}} and the
```

Comment:
I also find "`timestamp`s" a bit awkward to read.  But we need a plural here I think, right?  In that case, does "`timestamp` values" read better?

Whatever you chose, please consider making the same change everywhere in this CL.

Reply:
ah, good spot. yes, adding `values` should help here. will make that change